Firmware/Marlin_main.cpp

```diff
@@ -1382,10 +1357,6 @@ void setup()
 
 	setup_photpin();
 
-#if 0
-	servo_init();
```

Comment:
I'd also delete Servo.h/cpp. They are unusable on all boards that we support since most (and in some cases all) timers are used up for something else. That something else will surely break if Servos are attached. Better not have that old code anymore

Reply:
Done 👍 Hopefully I didn't miss anything. [eb79694](https://github.com/prusa3d/Prusa-Firmware/pull/4739/commits/eb7969441a01dd51afa3a0b59d79a289cb76ecd9)